app/validators/blocks/validate_collector_quesitons_mixin.py

```diff
@@ -0,0 +1,29 @@
+from app.validators.validator import Validator
+
+
+def _options_contain_value(options, value):
+    for option in options:
+        if option["value"] == value:
+            return True
+
+
+class ValidateCollectorQuestionsMixin(Validator):
+    def __init__(self, *args, **kwargs):
+        super(ValidateCollectorQuestionsMixin, self).__init__(*args, **kwargs)
+
+    def validate_collector_questions(
```

Comment:
Didn't understand this at first, can we rename so it says `ListCollector` instead of `Collector`, we only have 1 type.